EIPS/eip-0000.md

```diff
@@ -0,0 +1,183 @@
+---
+eip: 0000
+title: Backward compatible SWAPN, DUPN, EXCHANGE
+description: Introduce additional instructions for manipulating the stack which allow accessing the stack at higher depths
+author: Alex Beregszaszi (@axic), Charles Cooper (@charles-cooper), Francisco Giordano (@frangio)
+discussions-to: https://ethereum-magicians.org/t/eip-backward-compatible-swapn-dupn-exchange/25486
+status: Review
```

Comment:
Should the status revert back to draft?

Reply:
I don't think there's a formal requirement for that. The EIP is ready for review from my perspective.